apps/aecore/src/aec_dev_reward.erl

```diff
@@ -82,18 +102,47 @@ activated(Protocol) ->
     end.
 
 
-unparse_beneficiary({PK, Share}) ->
-    <<(aeser_api_encoder:encode(account_pubkey, PK))/binary, ":", (integer_to_binary(Share))/binary>>.
+unparse_beneficiary({PK, Share, FromProtocol, ToProtocol}) ->
```

Comment:
unparse... How about `encode`? 😅 But I know it is legacy.

Reply:
Yes, I prefer that, it is clearer.